Libplanet/Net/Swarm.cs

```diff
@@ -2163,6 +2025,116 @@ private void TransferRecentStates(GetRecentStates getRecentStates)
             Transport.ReplyMessage(reply);
         }
 
+        private async Task ProcessFillblock(CancellationToken cancellationToken)
+        {
+            while (!cancellationToken.IsCancellationRequested)
+            {
+                if (_demandBlockHash is null ||
+                    _demandBlockHash.Value.Item1 <= BlockChain.Tip.Index)
+                {
+                    await Task.Delay(100, cancellationToken);
+                }
+                else
+                {
+                    (long index, BoundPeer peer, HashDigest<SHA256> blockHash) =
+                        _demandBlockHash.Value;
+                    if (index > BlockChain.Tip.Index)
```

Comment:
Is this condition different from [the previous condition](https://github.com/planetarium/libplanet/pull/784/files#diff-9501da6c2e844a81d5e3ad3d3d5456ccR2032)?